### ISSUE TITLE
third user on notes document

### ISSUE THREAD

**Issue by johnabuck** [agilepro/cognoscenti#473]
strange bug happened when we were running meeting.

This msg showed up when I clicked on Roles

![image](https://user-images.githubusercontent.com/10082365/73687310-b6da7e80-4697-11ea-9bb3-5c8af407a1de.png)

Really weird. One person is typing in her set of the notes but it's not appearing on my screen: 
Her screen: 
![image](https://user-images.githubusercontent.com/10082365/73688905-0b332d80-469b-11ea-985c-3031f19112bb.png)

Here's what I see:
![image](https://user-images.githubusercontent.com/10082365/73688954-29009280-469b-11ea-9966-a7e07db49638.png)

It's like there are two separate instances of the meeting happening

When we tried to conclude the meeting, these notes by Carol did not appear on the main agenda document:

![image](https://user-images.githubusercontent.com/10082365/73693034-7bde4800-46a3-11ea-9500-b60cf8cee4fe.png)

I had her do a copy paste of each item so that we did not lose the notes

**Comment by agilepro** [comment]
I think it might be related to this problem I encountered:

![image](https://user-images.githubusercontent.com/5544889/74070905-2b266200-49b7-11ea-9b32-4628fefffde5.png)

**Comment by agilepro** [comment]
I want to be completely clear about this fix.   It had nothing to do with having three people editing.  There should be no problem having lots of people editing at the same time.

Instead, the problem was caused by a small network problem.   If a particularly web request FAILED, it would leave a flag variable in the wrong mode, and there was no way to recover.   I have cleaned up the logic so now this particular network error will not cause this problem.
